gfeloqua.class.php

```diff
@@ -98,8 +98,6 @@ public function init(){
 		// entry detail
 		add_action( 'gform_entry_detail', array( $this, 'entry_notes' ), 10, 2 );
 
-		// entry meta (for column)
-		add_action( 'gform_entry_meta', array( $this, 'add_success_meta' ), 10, 2 );
```

Comment:
It's been awhile since I've used this plugin on a live site. What's the reason for removing the `add_success_meta` functionality?